jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/jaxrs31/ee/multipart/MultipartSupportIT.java

```diff
@@ -0,0 +1,426 @@
+ * @author <a href="mailto:jckofbyron@gmail.com">Jim Krueger</a>
+ */
+
+// To-do:  Currently the find method is used to find EntityParts in a returned List because the order can not be 
+// guarenteed.   If this is addressed then tests should be added or modified.
+
+@ExtendWith(ArquillianExtension.class)
+@RunAsClient
+public class MultipartSupportIT {
+    private static final long serialVersionUID = 31L;
+
+    private static final String LS = System.lineSeparator();
+    static InputStream xmlFile() {
+        String xml =
+                "<root>" + LS +
+                "  <mid attr1=\"value1\" attr2=\"value2\">" + LS +
+                "    <inner attr3=\"value3\"/>" + LS +
+                "  </mid>" + LS +
+                "  <mid attr1=\"value4\" attr2=\"value5\">" + LS +
+                "    <inner attr3=\"value6\"/>" + LS +
+                "  </mid>" + LS +
+                "</root>";
+        return new ByteArrayInputStream(xml.getBytes());
+    }
+
+
+    @BeforeEach
+    void logStartTest(TestInfo testInfo) {
+        TestUtil.logMsg("STARTING TEST : "+testInfo.getDisplayName());
+    }
+
+    @AfterEach
+    void logFinishTest(TestInfo testInfo) {
+        TestUtil.logMsg("FINISHED TEST : "+testInfo.getDisplayName());
+    }
+
+    @Deployment
+    public static WebArchive deployment() {
+        return ShrinkWrap.create(WebArchive.class, MultipartSupportIT.class.getSimpleName() + ".war")
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+    }
+
+    @ArquillianResource
+    private URI uri;
+
+    /**
+     * Verify sending a List of {@link EntityPart} and receiving a List sent back from the resource method containing
+     * the same parts plus an additional part added by thye resource method containing extra headers.
+     *
+     * @throws Exception if an error occurs in the test
+     */
+    @Test
+    public void basicTest() throws Exception {
+        try (Client client = ClientBuilder.newClient()) {
+            final List<EntityPart> multipart = List.of(
+                    EntityPart.withName("octet-stream")
+                            .content("test string".getBytes(StandardCharsets.UTF_8))
+                            .mediaType(MediaType.APPLICATION_OCTET_STREAM_TYPE)
+                            .build(),
+                    EntityPart.withName("file")
+                            .content("test file", xmlFile())
+                            .mediaType(MediaType.APPLICATION_XML)
+                            .build()
+                    );
+            try (
+                    Response response = client.target(createCombinedUri(uri, "test/basicTest"))
+                            .request(MediaType.MULTIPART_FORM_DATA_TYPE)
+                            .post(Entity.entity(new GenericEntity<>(multipart) {
+                            }, MediaType.MULTIPART_FORM_DATA))) {
+                Assertions.assertEquals(Response.Status.OK, response.getStatusInfo());
```

Comment:
Again, `StatusType` is not equal to `Status`

Reply:
Agreed